package.json

```diff
@@ -114,6 +114,12 @@
                     "type": "number",
                     "default": 5000,
                     "description": "Controls the save delay in milliseconds."
+                },
+                "local-history.fileLimit": {
+                    "type": "number",
+                    "default": 30,
+                    "minimum": 1,
```

Comment:
Should we add a maximum?
Or perhaps it could be handled by the new global preference controlling the number of all revisions allowed (ex: the number of total files or space `.local-history` uses)

Reply:
We can create a new preference maintaining the maximum allowance of space, yes. 
Individually in this extension, I didn't understand what to keep as the maxima, it highly depends on the user.